Makefile

```diff
@@ -5,6 +5,7 @@ include common.mk
 
 DOCKER_IMAGE ?= azul
 DOCKER_TAG ?= latest
+IGNORE_CACHE ?= False
```

Comment:
`False` could be misconstrued by a developer think they need to set this to True. Can we use empty string as a default.